tests/data/test_spark_dataset.py

```diff
@@ -248,7 +261,7 @@ def test_from_spark_delta_table_name_and_version(spark_session, tmp_path, df):
     # write to delta table
     df_spark.write.format("delta").mode("overwrite").saveAsTable("my_delta_table")
 
-    mlflow_df = mlflow.data.from_spark(df_spark, table_name="my_delta_table", version=1)
```

Comment:
By writing to the table once, it's unclear why `version=1` should work...changing to `version=0`